src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -697,6 +660,8 @@ absl::flat_hash_set<ActorID> GcsActorManager::GetUnresolvedActorsByOwnerWorker(
 void GcsActorManager::OnWorkerDead(const ray::ClientID &node_id,
                                    const ray::WorkerID &worker_id,
                                    bool intentional_exit) {
+  RAY_LOG(WARNING) << "Worker " << worker_id << " on node " << node_id
```

Comment:
Isn't it supposed to be INFO actually?

Reply:
I think warning is better, because worker is dead.

Reply:
This should be `INFO`, it happens under normal operation. You can change it to warning if `intentional_exit` is false.

Reply:
Good idea, thanks.